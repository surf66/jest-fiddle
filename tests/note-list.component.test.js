import React from 'react';
import NoteList from '../src/js/components/note-list.component.js';

test('NoteList should display a list item for each note', () => {
  const notes = ['One', 'Two', 'Three'];
  const tree = shallow(<NoteList notes={notes} />);

  expect(tree).toMatchSnapshot();
});