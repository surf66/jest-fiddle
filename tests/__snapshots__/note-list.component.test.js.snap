// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteList should display a list item for each note 1`] = `
<ul>
  <li
    key="0"
  >
    One
  </li>
  <li
    key="1"
  >
    Two
  </li>
  <li
    key="2"
  >
    Three
  </li>
</ul>
`;
